deploy.sh

```diff
@@ -47,12 +46,15 @@ git checkout $SOURCE_BRANCH
 
 # Actual building and setup of current push or PR.
 npm install
-npm run predeploy
+npm run build
+mv build ../build/
+
 git checkout $TARGET_BRANCH
-mv build/* .
+rm -rf node_modules/
+mv ../build/* .
```

Comment:
When we do git add, this build folder gets added as well, so I moved it one directory up and then copied it's files back.